api/internal/target/kusttarget.go

```diff
@@ -155,11 +155,19 @@ func (kt *KustTarget) addHashesToNames(
 // not yet fixed.
 func (kt *KustTarget) AccumulateTarget() (
 	ra *accumulator.ResAccumulator, err error) {
-	ra = accumulator.MakeEmptyAccumulator()
-	err = kt.accumulateResources(ra, kt.kustomization.Resources)
+	return kt.accumulateTarget(accumulator.MakeEmptyAccumulator())
+}
+
+func (kt *KustTarget) accumulateTarget(ra *accumulator.ResAccumulator) (
```

Comment:
would you add a comment about how this is intended to be used -- I am able to understand from the code, but a comment explaining its purpose would be helpful.

Reply:
Brief comment added.  Is this enough?  I think that it would ultimately be better to make this behaviour more explicit via some refactoring, but that is probably another discussion? (e.g. by encapsulating the differences in behaviour between Kustomizations and Components in function objects or different classes)

Reply:
this is good for now.  thanks